in/index.md

```diff
@@ -6,29 +6,29 @@ title: Daftar Isi Bantuan
 
 <div class="panel panel-info">
   <div class="panel-heading">
-    <strong>You can't find a solution to your problem? You still have questions?</strong>
+    <strong>Kamu tidak menemukan solusi ketika ada masalah ? atau kamu masih mempunyai pertanyaan ?</strong>
   </div>
   <div class="panel-body">
-    <a class="btn btn-default" role="button" href="http://discourse.jabref.org">Use the online forum to get more support!</a>
+    <a class="btn btn-default" role="button" href="http://discourse.jabref.org">Gunakan forum online untuk mendapatkan beberapa bantuan!</a>
   </div>
 </div>
 
 
 ## FAQ
-- [Contributing](/in/FAQcontributing)
-- [General](/in/FAQgeneral)
+- [Kontribusi](/in/FAQcontributing)
+- [Umum](/in/FAQgeneral)
 - [JabRef and Linux](/in/FAQlinux)
 - [JabRef and Mac OS X](/in/FAQosx)
 - [JabRef and Windows](/in/FAQwindows)
-- [Other](/in/FAQother)
-- [Sharing](/in/FAQsharing)
+- [Lainnya](/in/FAQother)
+- [Berbagi](/in/FAQsharing)
 
 
 ## Umum
 - [Backup](/in/Backup)
 - [Best Practices](/in/BestPractices)
 - [Command line use and options](/in/CommandLine)
-- [Installation](/in/Installation)
```

Comment:
Translation does not work like that in this tool.

Please go into https://github.com/JabRef/help.jabref.org/tree/gh-pages/in and place translated files for https://github.com/JabRef/help.jabref.org/tree/gh-pages/en

For instance, for "Installation", translate https://github.com/JabRef/help.jabref.org/blob/gh-pages/en/Installation.md into https://github.com/JabRef/help.jabref.org/blob/gh-pages/in/Installation.md